File: java/fizzbuzz.java
import java.util.*;
	public class FizzBuzz
	{
		public static void main (String [] args)
		{
			int i=1;
			while (i<=100)
			{if (i%3==0 && i%5==0)
				System.out.printf ("FizzBuzz\n");
			else if (i%3==0)
				System.out.printf ("Fizz\n");
			else if (i%5==0)
				System.out.printf ("Buzz\n");
			else System.out.printf ("%d\n", i);
				i++;}
		}
	}

File: nim/fizzbuzz.nim
proc fizzbuzz() =
  for i in 1..100:
    if (i %% 3 == 0) and (i %% 5 == 0):
      echo "FizzBuzz"
    elif i %% 3 == 0:
      echo "Fizz"
    elif i %% 5 == 0:
      echo "Buzz":
    else:
      echo i  

fizzbuzz()